force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +325,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
```

Comment:
(s) Solid logic. However, there is a [list method](https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_list.htm) that will do all this work for you. Can you find it and refactor your code to use it?